cdf-core/cdf/js-modules/components/filter/base/Logger.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * @module BaseFilter
+ */
+
+/**
+ * Mixin that provides console logging abilities
+ * @class Logger
+ * @static
+ * @extensionfor AbstractSelect
+ * @extensionfor Models.Tree
+ * @extensionfor Views.Abstract
+ * @main
+ */
+
+ define([
+    'cdf/Logger',
+    './BaseFilter'],
+    function( Logger, BaseFilter ) {
+
+        BaseFilter.Logger = {
+
```

Comment:
Shouldn't this be BaseFilter.Logger = Logger; ?

Why are we repeating the code for Logger here ? 

I'm mostly ok with having this mixin extending an object with logging method definitions (although I'd rename it to LoggerExtension or LoggerMixin or something similar). Not ok with repeating the same lines of code that exist in cdf/Logger.